core/planner.c

```diff
@@ -1080,6 +1080,13 @@ bool plan(struct deco_state *ds, struct diveplan *diveplan, struct dive *dive, i
 		diveplan->eff_gflow = lrint(100.0 * (regressiona() * first_stop_depth + regressionb()));
 	}
 
+	for (int i = 0; i < MAX_CYLINDERS; i++)
+		if (cylinder_nodata(&dive->cylinder[i])) {
+			// Switch to an empty air cylinder for breathing air at the surface
+			current_cylinder = i;
+			break;
+		}
+	plan_add_segment(diveplan, 600, 0, current_cylinder, 0, false, OC);
```

Comment:
adding this unconditional, instead of as an option. Hmm. Interesting. I didn't expect that.

Reply:
UI is added in second commit as you will see. This first commit is to get the logic right.